service_mesh/v2x/ossm-gateway-migration.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-gateway-migration"]
+= Gateway Migration
+include::_attributes/common-attributes.adoc[]
+:context: gateway-migration
+
+toc::[]
+
+As a network administrator, the preferred method for deploying ingress and egress gateways is to do so with a `Deployment` resource using gateway injection.
```

Comment:
Nit - do you think the sentence still means the same with these words trimmed out?
```suggestion
As a network administrator, the preferred method for deploying ingress and egress gateways is with a `Deployment` resource using gateway injection.
```

Reply:
done